bgpd/bgp_vty.c

```diff
@@ -3600,6 +3600,57 @@ DEFUN (bgp_default_shutdown,
 	return CMD_SUCCESS;
 }
 
+DEFPY(bgp_shutdown_msg, bgp_shutdown_msg_cmd, "bgp shutdown message MSG...",
+      BGP_STR
+      "Enable administrative shutdown of the BGP instance\n"
+      "Add a shutdown message (RFC 8203)\n"
+      "Shutdown message\n")
+{
+	char *msgstr = NULL;
+
+	VTY_DECLVAR_CONTEXT(bgp, bgp);
+
+	if (argc > 3)
+		msgstr = argv_concat(argv, argc, 3);
+
+	bgp_shutdown_enable(bgp, msgstr);
+	XFREE(MTYPE_TMP, msgstr);
+
+	return CMD_SUCCESS;
+}
+
+DEFPY(bgp_shutdown, bgp_shutdown_cmd, "bgp shutdown",
+      BGP_STR "Enable administrative shutdown of the BGP instance\n")
+{
+	VTY_DECLVAR_CONTEXT(bgp, bgp);
+
+	bgp_shutdown_enable(bgp, NULL);
+
+	return CMD_SUCCESS;
+}
+
+DEFPY(no_bgp_shutdown, no_bgp_shutdown_cmd, "no bgp shutdown",
+      NO_STR BGP_STR "Disable administrative shutdown of the BGP instance\n")
```

Comment:
I'm just curious about the description for this command. When I type `no bgp ?` I would see `Disable administrative...`, for someone might be confusion like _No disable administrative_ :)

Reply:
After a short discussion with @mwinter-osr, i changed the wording from "Disable" to "Remove". See [my latest commit](https://github.com/FRRouting/frr/pull/6938/commits/dc5291cbc726eaee5df44a01d129dc6697ad59d4) for reference.